app/main.py

```diff
@@ -1,6 +1,15 @@
+import requests
+import os
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.environ["API_KEY"]
+    print(f"Running with API key: {api_key}, weather in Paris now (JSON):")
+    weather_in_paris_current = requests.get(
+        "http://api.weatherapi.com/v1/current.json?"
+        f"key={api_key}&q=Paris"
+    )
```

Comment:
Did you check the checklist? :)
https://github.com/mate-academy/py-docker-weather-api/blob/master/checklist.md#1-use-constants